exam_12.js

```diff
@@ -0,0 +1,74 @@
+// ex 1: 
+const checkArrayComparison = (arr1 = [], arr2 = []) => {
+  arr1.sort()
+  arr2.sort()
+  if (arr1.length == arr2.length && JSON.stringify(arr1) === JSON.stringify(arr2)) return true
+  else return false
+}
+
+// method2 - chưa hoàn thành code.
+/* 1.tạo hàm inputAray(arr) riêng -> return all=[arr1,arr2,arr3...] 
+2.khởi tạo hàm checkArrayComparison(all)
+3. tạo vòng for sắp xếp tuwnngf array trong all với sort()
+4. xét all[i] và all[j] vs (j =i+1) với điều kiện 
+if (all[i].length == all[j].length && JSON.stringify(all[i]) === JSON.stringify(all[j])
+và đưa ra kết quả true /false*/
+
+// ex 2:
+const result = []
+const flattenArray = (arr = []) => {
+  for (let i = 0; i < arr.length; i++) {
+    if (Array.isArray(arr[i]) && arr[i].length > 0) {
+      flattenArray(arr[i])
+    } else {
+      result.push(arr[i])
+    }
+  }
+  return result
+}
+
+// ex 3: 
+const chunks = (data, chunk) => {
+  const result = []
+  for (let i = 0; i < data.length; i += chunk) {
+    result.push(data.slice(i, i + chunk))
+  }
+  return result
+}
+
+// ex 4:
+//method1
```

Comment:
```suggestion
// Method 1
```

Mà `method1` là cái gì???